### PR TITLE
RSpec Supervisor Subject controller

### PR DESCRIPTION
**Pull request by shakil-fg**

### CODE REVIEW

**Comment by mdmahmudurrahman** [comment]
Okey.

File: Gemfile [hunk 1]
```diff
@@ -38,7 +38,7 @@ gem "whenever", require: false
 #gem "paranoia", "~> 2.0"
 
 gem "bcrypt",                  "3.1.7"
-#gem 'faker',                   '1.4.2'
+gem 'faker'
```
Comment (dieunb) [inline Gemfile]: double quote psl l

File: spec/controllers/supervisor/subjects_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,141 @@
+require 'rails_helper'
+
+describe Supervisor::SubjectsController do
+  describe 'GET #index' do
```
Comment (dieunb) [inline spec/controllers/supervisor/subjects_controller_spec.rb]: Double quote

**Comment by noor-framgia** [comment]
ok

**Comment by manazirahsan** [comment]
okey

**Pull request merged**